### PR TITLE
Fabo/1032 event tracking

### LINKED ISSUE #1032: What should we track?
UI Version: 0.8.2

Description:

What events should we track in google analytics?

Remove opt-in in favor of opt-out.

### LINKED ISSUE #1888: google analytics event setup
set up event tracking in google analytics

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1032 
Closes #1888 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/config.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const faucet = process.env.FAUCET || `https://faucet.voyager.ninja`
 
 export default {
   name: `Cosmos Voyager`,
+  development: dev,
```
Comment (jbibla) [inline app/src/config.js]: would boolean be better?

Comment (faboweb) [reply app/src/config.js]: it is a boolean as far as I know

Comment (faboweb) [reply app/src/config.js]: see line 1

Comment (jbibla) [reply app/src/config.js]: ah apologies.

**Pull request merged**